src/extension.ts

```diff
@@ -5,6 +5,7 @@ import { startLanguageServer } from "./commands/start-lsp";
 import { stopLanguageServer } from "./commands/stop-lsp";
 import { log } from "./log";
 import { activateStatusBar, deactivateStatusBar } from "./status-bar";
+import { bufState } from "./state";
```

Comment:
This import should be unused now ^^" Hmm, unfortunate that CI doesn't catch that... o_o